src/app/visual/single-file-attachment/single-file-attachment-visual.component.ts

```diff
@@ -44,7 +45,7 @@ export class SingleFileAttachmentVisualComponent implements OnInit {
   }
 
   public ngOnInit(): void {
-    this.attachment = new FormControl(undefined, Validators.required);
```

Comment:
☝️ Even though I removed the reactive validator here in favor of using the new `@Input required` property, this is still supported. See line `159` of `file-attachment.component.ts `.